test/tools/yulInterpreter/Interpreter.h

```diff
@@ -183,6 +183,9 @@ class Interpreter: public ASTWalker
 		m_disableExternalCalls(_disableExternalCalls),
 		m_disableMemoryTrace(_disableMemoryTracing)
 	{
+		auto const* evmDialect = dynamic_cast<EVMDialect const*>(&_dialect);
+		// TODO: Add EOF support
+		yulAssert(evmDialect && !evmDialect->eofVersion());
```

Comment:
This should be `solUnimplemented()`. And I'd move it to `yulrun`. We should safeguard the entry point, just like we did it with EOF in various `solc` features, but the internal implementation should still remain testable on EOF.

Reply:
Done!